### PR TITLE
Self service GCP

### PR DESCRIPTION
**Pull request by Deflaimun**
Fixes https://github.com/redpanda-data/documentation-private/issues/1817

preview:
- https://deploy-preview-22--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/
- https://deploy-preview-22--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/manage-billing/
- https://deploy-preview-22--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/manage-billing/billing/
- https://deploy-preview-22--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/manage-billing/gcp-pay-as-you-go/
- https://deploy-preview-22--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/manage-billing/gcp-commit/

### CODE REVIEW

File: modules/deploy/pages/deployment-option/cloud/manage-billing/billing.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
+= Marketplace Consumption Metrics
```
Comment (Deflaimun) [inline modules/deploy/pages/deployment-option/cloud/manage-billing/billing.adoc:1]:
```suggestion
= Marketplace Consumption Metrics 
```

File: modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-pay-as-you-go.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+. On the Redpanda page: 
+* For **Email**, enter your email address to register with Redpanda.
+* For **Organization name**, enter a name for your new organization connected through Google Cloud Marketplace. Redpanda organizations contain all resources, including clusters and networks. You can rename the organization name later in Redpanda Console. 
+* Click **Sign up and create organization**.
+
+. On the email sent to the address you entered, click **Verify email address**. 
++
+This completes the registration and associates the email with a Redpanda account. 
+
+. On the **Accept your invitation to sign up** page, click **Sign up** or **Log in**.
```
Comment (micheleRP) [inline modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-pay-as-you-go.adoc:37]: @simon0191 should this be sign up or login?

File: modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-pay-as-you-go.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+If you don't have a billing account associated with your project, you're prompted to enable billing to link the subscription with a billing account.
+====
+
+. On the Order Summary page, the **Pay As You Go** plan using Redpanda consumption units is automatically selected. 
+* Read and accept the terms.
+* Click **Subscribe**.
+
+. On the Redpanda page: 
+* For **Email**, enter your email address to register with Redpanda.
+* For **Organization name**, enter a name for your new organization connected through Google Cloud Marketplace. Redpanda organizations contain all resources, including clusters and networks. You can rename the organization name later in Redpanda Console.
```
Comment (micheleRP) [inline modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-pay-as-you-go.adoc]: @c4milo can we change the organization name in Cloud?

File: modules/deploy/pages/deployment-option/cloud/manage-billing/billing.adoc [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+Redpanda Cloud charges for all data transferred each hour, including each produce and consume requests.
+
+* The cost for an ingress unit depends on the region you selected for your Dedicated cluster. Each ingress unit equals one GB of data written. 
+* The cost for an egress unit depends on the number of availability zones (AZ) you selected for your Dedicated cluster. Each egress unit equals one GB of data read.
+
+=== Storage metrics
+
+Redpanda Cloud charges for the volume of data stored on a cluster, including replication. Each storage unit equals one GB of data in cloud storage per hour. 
+
+Replication is important for high availability. Because Redpanda Cloud has a default replication factor of three, the amount of data stored typically exceeds the volume of data written to the cluster. To reduce storage usage, you can configure the `retention.bytes` and `retention.ms` on a topic. See xref:reference:topic-properties.adoc[Topic Properties].
```
Comment (micheleRP) [inline modules/deploy/pages/deployment-option/cloud/manage-billing/billing.adoc]: @c4milo is this configurable in cloud?

File: modules/deploy/pages/deployment-option/cloud/manage-billing/billing.adoc [hunk 5]
```diff
@@ -0,0 +1,33 @@
+= Marketplace Consumption Metrics
+:description: Learn about the consumption units in Redpanda Cloud billing.
+
+When you subscribe to Redpanda Cloud through Google Cloud Marketplace, you can use your existing GCP billing and credits to provision Redpanda Dedicated Cloud clusters. View your bills and manage your subscription directly in Google Cloud Marketplace. You can either pay as you go or purchase through committed use.
```
Comment (alenkacz) [inline modules/deploy/pages/deployment-option/cloud/manage-billing/billing.adoc:4]:
```
purchase through committed use.
```

is this really open for people? I thought it's just private offers made by sales team

File: modules/deploy/pages/deployment-option/cloud/manage-billing/billing.adoc [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+=== Ingress and egress meters
+
+Redpanda Cloud charges for all data transferred each hour, including each produce and consume request.
+
+* Each ingress unit equals one GB/hour of data written. The cost for an ingress unit depends on the region you selected for your cluster. 
+* Each egress unit equals one GB/hour of data read. The cost for an egress unit depends on the number of availability zones (AZ) you selected for your cluster. 
+
+=== Storage meters
+
+Each storage unit equals one GB of data in cloud storage. Redpanda Cloud charges for the volume of data stored on a cluster, including replication.
```
Comment (alenkacz) [inline modules/deploy/pages/deployment-option/cloud/manage-billing/billing.adoc]: oh is it true? if you have the data with replication=3 we will store it 3 times in S3? I was expecting some deduplication but that said - I don't know :)

File: modules/deploy/pages/deployment-option/cloud/manage-billing/billing.adoc (hunk 4 above)
Comment (alenkacz) [reply modules/deploy/pages/deployment-option/cloud/manage-billing/billing.adoc]: yes

File: modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-commit.adoc [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+[NOTE]
+====
+If you don't have a billing account associated with your project, you're prompted to enable billing to link the subscription with a billing account.
+====
++
+You are taken to the Redpanda sign-up page.
+
+. On the Redpanda sign-up page: 
+* For **Email**, enter your email address to register with Redpanda.
+* For **Organization name**, enter a name for your new organization connected through Google Cloud Marketplace. Redpanda organizations contain all resources, including clusters and networks. You can rename the organization name later in Redpanda Console.
```
Comment (alenkacz) [inline modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-commit.adoc]: is it really possible to rename org? And through console? I thought console does not really interact with organizations

File: modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-pay-as-you-go.adoc (hunk 3 above)
Comment (alenkacz) [reply modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-pay-as-you-go.adoc]: I think not, I don't see it ... but will wait for someone from UI to confirm

**Review by alenkacz — COMMENTED** [review]
Left few comments

File: modules/deploy/pages/deployment-option/cloud/manage-billing/billing.adoc (hunk 6 above)
Comment (alenkacz) [reply modules/deploy/pages/deployment-option/cloud/manage-billing/billing.adoc]: I think this really is not true - reference https://redpandadata.slack.com/archives/C01ND4SVB6Z/p1694511963692429

File: modules/deploy/pages/deployment-option/cloud/manage-billing/billing.adoc (hunk 5 above)
Comment (simon0191) [reply modules/deploy/pages/deployment-option/cloud/manage-billing/billing.adoc:4]: cc @towfiqa

File: modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-pay-as-you-go.adoc (hunk 3 above)
Comment (simon0191) [reply modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-pay-as-you-go.adoc]: cc @malinskibeniamin @yougotashovel

File: modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-pay-as-you-go.adoc (hunk 2 above)
Comment (simon0191) [reply modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-pay-as-you-go.adoc:37]: You can do both. I think the explanation makes sense as is.

File: modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-pay-as-you-go.adoc (hunk 3 above)
Comment (yougotashovel) [reply modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-pay-as-you-go.adoc]: We can manually in Auth0, but customers cannot.  Not in our roadmap but it's technically plausible.

Comment (micheleRP) [reply modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-pay-as-you-go.adoc]: Removing: "You can rename the organization later in Redpanda Console."

File: modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-commit.adoc (hunk 7 above)
Comment (micheleRP) [reply modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-commit.adoc]: Removed "You can rename the organization name later in Redpanda Console."

**Comment by micheleRP** [comment]
Per Towfiqa, no docs necessary at this time.

**Pull request closed without merging**